adapters/tappx/tappx.go

```diff
@@ -137,7 +131,15 @@ func (a *TappxAdapter) buildEndpointURL(params *openrtb_ext.ExtImpTappx, test in
 		}
 	}
 
-	endpointParams := macros.EndpointTemplateParams{Host: params.Host}
+	tappxHost := "tappx.com"
+	isNewEndpoint, err := regexp.Match(`^(zz|vz)[0-9]{3,}([a-z]{2}|test)$`, []byte(params.Endpoint))
+	if isNewEndpoint {
+		tappxHost = params.Endpoint + ".pub." + tappxHost + "/rtb/"
```

Comment:
@prebidtappx Quick question: Is the `params.Endpoint + ".pub." + tappxHost + "/rtb/"` going to be the new Tappx endpoint and is the plan to deprecate `tappxHost = "ssp.api." + tappxHost + "/rtb/v2/"` in the future?

Reply:
Yes, it is the plan. However, a lot of people don't like the changes. For this reason, all the changes are backward compatible.